### PR TITLE
add space65 macOS keymap for vim users with an optimized bottom row

### PR DESCRIPTION
**Pull request by keithlo**
Added a new keymap for Space65 with a split space row compatibility. There were no previous maps that included this functionality

## Description
simple keymap with ascii art and description of how to flash the board. None of the other models had info that was necessary for me to flash the v1 board from the factory

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gray_studio/space65/keymaps/keithlo/keymap.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 MechMerlin
```
Comment (fauxpark) [inline keyboards/gray_studio/space65/keymaps/keithlo/keymap.c:1]: You should probably change the copyright here :)

File: keyboards/gray_studio/space65/keymaps/keithlo/keymap.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   * | Ctrl |  Alt  |  Cmd |    Space   |  Fn  |      Cmd     |  Alt |  Fn  |    | Left | Down |Right |
+   * `----------------------------------------------------------------------'    '--------------------'
+   */
+
+[0] = LAYOUT( \
+      KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_DEL,  \
+      KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_INS,  \
+      KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,           KC_PGUP, \
+      KC_LSFT, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_PGDN, \
+      KC_LCTL, KC_LALT, KC_LGUI,                 KC_SPC,   MO(1),          KC_RGUI,            KC_RALT,    MO(1),          KC_LEFT, KC_DOWN, KC_RGHT  \
```
Comment (fauxpark) [inline keyboards/gray_studio/space65/keymaps/keithlo/keymap.c:39]:
```suggestion
[0] = LAYOUT(
      KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_DEL,
      KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_INS,
      KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,           KC_PGUP,
      KC_LSFT, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_PGDN,
      KC_LCTL, KC_LALT, KC_LGUI,                 KC_SPC,   MO(1),          KC_RGUI,            KC_RALT,    MO(1),          KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/gray_studio/space65/keymaps/keithlo/keymap.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+   * |      |       |      |            |      |              |     |      |     | Home | PgDn |  End |
+   * `---------------------------------------------------------------------'     '--------------------'
+   */
+
+[1] = LAYOUT( \
+      KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_VOLD, KC_VOLU, KC_MUTE, KC_MUTE,  KC_MUTE, \
+			KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, RGB_SPD,  RGB_SPI, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          \
+			KC_TRNS, RGB_M_P, RGB_M_G, RGB_M_K, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT, KC_TRNS, KC_TRNS, KC_MPLY, KC_TRNS,                   \
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BRID,  KC_BRIU, KC_TRNS, KC_TRNS, KC_PGUP, KC_TRNS,          \
+			KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,  KC_TRNS,          KC_HOME, KC_PGDN, KC_END            \
```
Comment (fauxpark) [inline keyboards/gray_studio/space65/keymaps/keithlo/keymap.c:61]:
```suggestion
[1] = LAYOUT(
      KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_VOLD, KC_VOLU, KC_MUTE, KC_MUTE,  KC_MUTE,
			KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, RGB_SPD,  RGB_SPI, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
			KC_TRNS, RGB_M_P, RGB_M_G, RGB_M_K, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT, KC_TRNS, KC_TRNS, KC_MPLY, KC_TRNS,
			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BRID,  KC_BRIU, KC_TRNS, KC_TRNS, KC_PGUP, KC_TRNS,
			KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,  KC_TRNS,          KC_HOME, KC_PGDN, KC_END
```

File: keyboards/gray_studio/space65/keymaps/keithlo/keymap.c (hunk 1 above)
Comment (keithlo) [reply keyboards/gray_studio/space65/keymaps/keithlo/keymap.c:1]: will do :)

**Comment by drashna** [comment]
There are some merge conflicts here.

**Pull request closed without merging**